scripts/helm/helm_handle_chart.sh

```diff
@@ -58,6 +58,9 @@ else
         KUBE_CONFIG_FILE="$KUBE_CONFIG_PATH"
         KUBECONFIG="$KUBE_CONFIG_FILE"
         export KUBECONFIG=$KUBECONFIG:$KUBE_CONFIG_FILE
+    elif [[ -f "$KUBE_CONFIG_FILE" ]]; then
+        echo "KUBE_CONFIG_FILE ($KUBE_CONFIG_FILE) exists."
+        export KUBECONFIG="$KUBE_CONFIG_FILE"
```

Comment:
I notice in all other instances, you are extending the `KUBCONFIG` env var:
```
export KUBECONFIG=$KUBECONFIG:$KUBE_CONFIG_FILE
```
Here you are only setting it. Is that intentional?

Reply:
yes.  the idea in this case is that `KUBECONFIG` comes from `KUBE_CONFIG_FILE` (because we just checked for it), so no need to default to `KUBECONFIG` 

I'm not sure we need to be using the colon operator in the other instances, tbh.. lol